src/ImageSharp/Image/IImage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using SixLabors.ImageSharp.Formats;
+using SixLabors.ImageSharp.MetaData;
+
+namespace SixLabors.ImageSharp
+{
+    /// <summary>
+    /// Represents the base image abstraction.
+    /// </summary>
+    public interface IImage
+    {
+        /// <summary>
+        /// Gets information about pixel.
```

Comment:
Lets flesh out the property descriptors here. This doesn't tell me anything and wont help with docs.

Reply:
Done in 61c9caf.